index.html

```diff
@@ -47,11 +47,11 @@
           </ul>
         </div>
 
-        <button>Donate Now</button>
+        <a class="button donation" href="#">Donate Now</a>
```

Comment:
Interesting that you chose to use a link (anchor element) for the buttons. Links should indeed be used for navigating to another page, however it's confusing to users if a link is styled to look like a button. So in this case, I think the original design could be improved :)

Reply:
I did it just to show it supposed to be a link :)